quiz.js

```diff
@@ -180,5 +185,22 @@ function startQuiz() {
   startTimer();
 }
 
+function showResultOnTimeOut() {
```

Comment:
Ta funkcja jest w porządku. Optymalizacja, którą można by było zrobić to połączyć showResultOnTimeOut() i showResult() bo w gruncie rzeczy są podobne. Unikniemy wtedy duplikacji kodu. Byłaby jedna funkcja ale wywoływana z parametrem, np showResult(reason).